service/qos/service_level_controller.hh

```diff
@@ -344,6 +346,15 @@ public:
      */
     void maybe_start_legacy_update_from_distributed_data(std::function<steady_clock_type::duration()> interval_f, service::storage_service& storage_service, service::raft_group0_client& group0_client);
 
+    /**
+     * Get mutations required to create `sl:driver` and remember it in `system.scylla_local`
```

Comment:
Why this ` and remember it in `system.scylla_local`` part? This function is not doing that.